cmake/ci.cmake

```diff
@@ -551,6 +551,20 @@ add_custom_target(ci_test_diagnostics
     COMMENT "Compile and test with improved diagnostics enabled"
 )
 
+###############################################################################
+# Enable diagnostic positions support.
+###############################################################################
+
+add_custom_target(ci_test_diagnostic_positions
```

Comment:
`ci_test_diagnostic_positions` is never called in the CI. Please add this to the list in https://github.com/nlohmann/json/blob/develop/.github/workflows/ubuntu.yml#L111.

Reply:
sorry I didn't know that, thanks for letting me know!